express/services/playerTeamSeasons.js

```diff
@@ -0,0 +1,16 @@
+const models = require('../../sequelize/models');
+
+function getAll(playerId) {
```

Comment:
1. IMHO, `getAll` should return the whole list of `playerTeamSeasons`. Can we give a more descriptive name to this function? Something like `getAllByPlayerId` or something like this?
2. Should we handle somehow the case when `playerId` is undefined?